AI/BattleAI/PotentialTargets.cpp

```diff
@@ -112,10 +110,10 @@ int64_t PotentialTargets::bestActionValue() const
 	return bestAction().attackValue();
 }
 
-AttackPossibility PotentialTargets::bestAction() const
+const AttackPossibility & PotentialTargets::bestAction() const
 {
 	if(possibleAttacks.empty())
 		throw std::runtime_error("No best action, since we don't have any actions");
-	return possibleAttacks[0];
+	return possibleAttacks.at(0);
```

Comment:
No need to use .at because there is check for empty above

Reply:
Can I use [] in const method? So it worked somehow before.

Reply:
yes, it has const version